### PR TITLE
UI :- Converted html table to Antd table for Dashboard and PipelinesVersion

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue :- #7367
- Converted html table to Antd table for Dashboard and Pipelines Version

#
### Type of change :

- [x] Improvement

#
### Frontend Screenshot

image
image
image
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx [hunk 1]
```diff
@@ -481,6 +480,114 @@ const DashboardDetails = ({
...
+      ),
+    },
+    {
+      title: 'Tags',
+      dataIndex: 'tags',
+      key: 'tags',
+      width: 300,
+      render: (text, record, index) => (
+        <div
+          className="tw-group tw-relative tableBody-cell"
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx:557]: Let's remove tailwind as well

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineVersion/PipelineVersion.component.tsx [hunk 2]
```diff
@@ -246,74 +288,15 @@ const PipelineVersion: FC<PipelineVersionProp> = ({
...
-                        )}
-                      </tbody>
-                    </table>
+                  <div className="m-y-md tw-col-span-full">
+                    <Table
+                      columns={tableColumn}
+                      data-testid="schema-table"
+                      dataSource={(currentVersionData as Pipeline)?.tasks}
+                      pagination={false}
+                      rowKey="name"
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineVersion/PipelineVersion.component.tsx:297]: Can you check is name gonna be unique?

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/PipelineVersion/PipelineVersion.component.tsx:297]: yes, it going to be unique.

File: openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx [hunk 3]
```diff
@@ -481,6 +480,114 @@ const DashboardDetails = ({
...
 
+  const tableColumn: ColumnsType<ChartType> = [
+    {
+      title: ' Chart Name',
+      dataIndex: 'chartName',
+      key: 'chartName',
+      width: 200,
+      render: (_, record) => (
+        <Link target="_blank" to={{ pathname: record.chartUrl }}>
+          <span className="tw-flex">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx:491]: Use `<Space>` here

File: openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx [hunk 4]
```diff
@@ -481,6 +480,114 @@ const DashboardDetails = ({
...
+      key: 'chartType',
+      width: 100,
+    },
+    {
+      title: 'Description',
+      dataIndex: 'description',
+      key: 'description',
+      width: 300,
+      render: (text, record, index) => (
+        <div className="tw-inline-block">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx:517]:
```suggestion
        <Space >
```

File: openmetadata-ui/src/main/resources/ui/src/components/DashboardVersion/DashboardVersion.component.tsx [hunk 5]
```diff
@@ -212,6 +214,51 @@ const DashboardVersion: FC<DashboardVersionProp> = ({
...
+            <span className="tw-flex">
+              <span className="tw-mr-1">{record.displayName}</span>
+              <SVGIcons
+                alt="external-link"
+                className="tw-align-middle"
+                icon="external-link"
+                width="16px"
+              />
+            </span>
+          </Link>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/DashboardVersion/DashboardVersion.component.tsx:234]: Button type of Link will work here

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineVersion/PipelineVersion.component.tsx [hunk 6]
```diff
@@ -212,6 +214,46 @@ const PipelineVersion: FC<PipelineVersionProp> = ({
...
   }, [currentVersionData]);
 
+  const tableColumn: ColumnsType<Task> = useMemo(
+    () => [
+      {
+        title: 'Task Name',
+        dataIndex: 'displayName',
+        key: 'displayName',
+        render: (text, record) => (
+          <Link target="_blank" to={{ pathname: record.taskUrl }}>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineVersion/PipelineVersion.component.tsx:224]: Same as above

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less [hunk 7]
```diff
@@ -16,6 +16,14 @@
...
 @primary-light: rgb(244, 240, 253);
 
+// Position
+.relative {
+  position: relative;
+}
+.absolute {
+  position: absolute;
+}
+
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/app.less:26]: Move this to `position.less`

**Comment by Ashish8689** [comment]
@chirag-madlani  changes has been made as per comments!

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**